Specs/DateTime/Dutch/TimePeriodExtractor.json

```diff
@@ -269,20 +249,18 @@
         "Length": 11
       }
     ],
-    "NotSupported": "dotNet",
     "NotSupportedByDesign": "javascript,python,java"
   },
   {
-    "Input": "Ik ben weg 20 minuten tot drie voor acht ʼs avonds",
```

Comment:
@iMicknl ?

Reply:
It looks like the crowdsourced specs for date/time are not very high quality. This case doesn't make sense to me either.